src/jllvm/vm/native/Lang.cpp

```diff
@@ -1 +1,74 @@
 #include "Lang.hpp"
+void jllvm::lang::SystemModel::arraycopy(jllvm::VirtualMachine&, jllvm::GCRootRef<jllvm::ClassObject>,
+                                         jllvm::GCRootRef<Object> src, std::int32_t srcPos,
+                                         jllvm::GCRootRef<Object> dest, std::int32_t destPos, std::int32_t length)
+{
+    if (!src || !dest)
+    {
+        llvm_unreachable("Not yet implemented"); // TODO throw NullPointerException
```

Comment:
Write the todos as `// TODO: <text>` above the `llvm_unreachable` (our comments are usually above corresponding lines, not on the line)